com.irccloud.desktop.json

```diff
@@ -0,0 +1,115 @@
+{
+  "app-id": "com.irccloud.desktop",
```

Comment:
`.desktop` was historically problematic, maybe just avoid it?

Reply:
I think it's well known enough now that the issues have been shaken out (e.g. Telegram is popular) but I don't mind changing - it really depends what/if is used upstream.

Reply:
Yeah, I assumed it generally works these days. I can change that if upstream chimes in.